LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.js

```diff
@@ -72,7 +72,6 @@ promise_test(async () => {
       "Promise rejects with a DOMException for abortion");
   assert_equals(rejection.name, "AbortError",
       "Rejected with 'AbortError' DOMException");
-  assert_equals(rejection.message, "signal is aborted without reason");
```

Comment:
Let's make sure to upstream this change to WPT. I agree enforcing a particular error message is not a good idea.

Reply:
I sycned the file into this PR